### ISSUE TITLE
Edit on Github button broken

### ISSUE THREAD

**Issue by jessicajzhang03** [ansible/ansible#73678]
##### SUMMARY

Clicking "Edit on Github" from the link https://docs.ansible.com/ansible/latest/index.html tries to edit the nonexistent file index.html in the rst folder (it should be going to one of the index files in the rst folder). I'm afraid I don't really know how rst works, or how it gets converted to html, so I'm not totally sure how to fix it.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

breadcrumbs.html
incidental.py

##### ANSIBLE VERSION

```paste below

```

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT

OS: MacOS 11.1
Browser: Chrome Version 88.0.4324.150 (Official Build) (x86_64)

**Comment by ansibot** [comment]
Files identified in the description:
* [`docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html`](https://github.com/ansible/ansible/blob/devel/docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html)
* [`hacking/shippable/incidental.py`](https://github.com/ansible/ansible/blob/devel/hacking/shippable/incidental.py)

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by Saurabh-Thakre** [comment]
We've two different files `intro_installation.rst` and `intro_configuration.rst` which we point in `index.rst` for https://docs.ansible.com/ansible/latest/installation_guide/index.html

Actually, we take the data from `intro_installation.rst` and `intro_configuration.rst` in index.html you're referring to. 

Also I could see that the Edit On GitHub takes to exact place where it should take i.e here /docsite/rst/installation_guide/index.rst

**Comment by jessicajzhang03** [comment]
Sorry, I think I copied the wrong link. The issue was here: https://docs.ansible.com/ansible/latest/index.html. (And the same issue if we change the version in the dropdown menu on the left, e.g., with https://docs.ansible.com/ansible/2.10/index.html.) I think it should probably be going to core_index.rst or ansible_index.rst. I guess maybe 2.10_index.rst for the 2.10 version one?

Also, a minor issue I noticed (should I just open a new issue for this, or is it a bit unnecessary?) is that once I select 2.10 from the dropdown menu, the new dropdown seems to think that 2.10 is the latest, even though the original page https://docs.ansible.com/ansible/latest/index.html has version 3.
